app/lib/features/chat_ng/widgets/events/profile_update_event.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:acter/common/extensions/options.dart';
```

Comment:
I am not convinced this is something we should be doing, but instead leave to the flutter devs to best figure out.